### PR TITLE
+ Add URL path for mission

### PR DESCRIPTION
**Pull request by mckingho**

### CODE REVIEW

File: pages/in/bonus/index.vue [hunk 1]
```diff
@@ -75,7 +75,7 @@
                   :missions="getMissionList"
                   :is-loading="getIsFetchingMissions || !getIsFetchedMissions"
                   :empty-placeholder="$t('BonusPage.placeholder.emptyMission')"
-                  @click="onMissionClick" />
+                  @click="onMissionClick" :selected-mission="selectedMission"/>
```
Comment (nwingt) [inline pages/in/bonus/index.vue]: Please move `:selected-mission` above `@click`

File: pages/in/bonus/index.vue [hunk 2]
```diff
@@ -211,6 +212,11 @@ export default {
       user: '',
     };
   },
+  asyncData({ query }) {
+    return {
+      selectedMission: query.selectedMission,
```
Comment (williamchong007) [inline pages/in/bonus/index.vue]: not asyncData

File: components/Mission/List.vue [hunk 3]
```diff
@@ -203,6 +207,12 @@ export default {
       this.$nextTick(() => {
         this.$refs.list.scrollLeft = 0;
         this.updateScrollIndicator();
+        if (this.selectedMission) {
+          const mission = this.missions.find(m => m.id === this.selectedMission);
```
Comment (nwingt) [inline components/Mission/List.vue]: Just wrap this in selectedMission computed property

File: components/Mission/List.vue [hunk 4]
```diff
@@ -203,6 +207,9 @@ export default {
       this.$nextTick(() => {
         this.$refs.list.scrollLeft = 0;
         this.updateScrollIndicator();
+        if (this.selectedMission) {
```
Comment (nwingt) [inline components/Mission/List.vue]: should check `!isLoading`

**Pull request merged**